src/Microsoft.AspNetCore.Razor.Evolution/RazorIRPass.cs

```diff
@@ -1,87 +1,14 @@
-    /// of reasonable defaults.
-    /// </para>
-    /// <para>
-    /// The second phase of IR processing is directive classification. IR passes in this phase should interpret 
-    /// directives and processing them accordingly by transforming IR nodes or adding diagnostics to the IR. At
-    /// this time the document kind has been identified, so any directive that can't be applied should trigger
-    /// errors. If implementing a document kind that diverges from the standard structure of Razor documents
-    /// it may be necessary to reimplement processing of default directives.
-    /// </para>
-    /// <para>
-    /// The last phase of IR processing is lowering. IR passes in this phase perform some kind of transformation
-    /// on the IR that optimizes the generated code. The key distinction here is that information may be discarded 
-    /// during this phase.
-    /// </para>
-    /// <para>
-    /// Finally, the <see cref="IRazorCSharpLoweringPhase"/> transforms the IR document into generated C# code.
-    /// At this time any directives or IR constructs that cannot be understood by code generation will result
-    /// in an error.
-    /// </para>
-    /// </remarks>
     public static class RazorIRPass
     {
         /// <summary>
-        /// An <see cref="IRazorIRPass"/> that implements a document classifier should use this value as its
-        /// <see cref="IRazorIRPass.Order"/>.
-        /// </summary>               
-        public static readonly int DocumentClassifierOrder = 1100;
-
-        /// <summary>
-        /// <see cref="IRazorIRPass.Order"/> value used by the default document classifier.
-        /// </summary>
-        public static readonly int DefaultDocumentClassifierOrder = 1900;
-
-        /// <summary>
-        /// An <see cref="IRazorIRPass"/> that implements a directive classifier should use this value as its
-        /// <see cref="IRazorIRPass.Order"/>.
-        /// </summary>    
-        public static readonly int DirectiveClassifierOrder = 2100;
-
-        /// <summary>
-        /// <see cref="IRazorIRPass.Order"/> value used by the default directive classifier.
-        /// </summary>
-        public static readonly int DefaultDirectiveClassifierOrder = 2900;
-
-        /// <summary>
-        /// An <see cref="IRazorIRPass"/> that implements a lowering phase should use this value as its
-        /// <see cref="IRazorIRPass.Order"/>.
-        /// </summary>    
-        public static readonly int LoweringOrder = 4100;
-
-        /// <summary>
-        /// <see cref="IRazorIRPass.Order"/> value used by the default lowering phase.
+        /// The default implementation of the <see cref="IRazorEngineFeature"/>s that run in a
+        /// <see cref="IRazorEnginePhase"/> will use this value for its Order property.
         /// </summary>
-        public static readonly int DefaultLoweringOrder = 4900;
+        public static readonly int DefaultFeatureOrder = 1000;
```

Comment:
I think the naming is OK, but the docs don't capture the intent. 

We choose a number for things that should run after your code by default. That's the point